eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -72,6 +72,35 @@ jobs:
           eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
           eq(variables['isRollingBuild'], true))
 
+#
+# CoreCLR NativeAOT release build and additional libraries tests
+# Only when CoreCLR or library is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
+    - windows_arm64
+    jobParameters:
+      testGroup: innerloop
+      isSingleFile: true
+      nameSuffix: NativeAOT_Additional_Libs
+      buildArgs: -s clr.alljits+clr.tools+clr.nativeaotlibs+clr.nativeaotruntime+libs+libs.tests -c $(_BuildConfig) /p:TestNativeAot=true /p:TestNativeAotAdditionalLibs=true /p:ArchiveTests=true
```

Comment:
Is it standard to use both build parameters "TestNativeAot" and "TestNativeAotAdditionalLibs". I would have thought we'd just use the new one. Either way is fine with me, I'm just trying to learn what the norm is.

Reply:
I wanted the original property to make it clear that this is NativeAOT testing and the new one to provide additional information. But perhaps using 1 might be simpler